Functions/QualityControl/xASL_qc_CollectParameters.m

```diff
@@ -1,22 +1,23 @@
-function x = xASL_qc_CollectParameters(x, iSubject, ScanType)
+function x = xASL_qc_CollectParameters(x, iSubject, ScanType, iSession)
 %xASL_qc_CollectParameters Collect all parameters from structural & ASL, spread over the derivative folders
 %
-% FORMAT: x = xASL_qc_CollectParameters(x, iSubject, ScanType, CollectQCFunction)
+% FORMAT: x = xASL_qc_CollectParameters(x, iSubject, ScanType, CollectQCFunction [, iSession])
 %
 % INPUT:
 %   x                   - structure containing fields with all information required to run this submodule (REQUIRED)
 %   iSubject            - index of current subject (REQUIRED)
 %   ScanType            - string for ScanType, options = 'Structural' 'ASL' 'func' 'dwi' (REQUIRED)
+%   iSession            - index of current session (OPTIONAL, default 1, but RECOMMENDED for ScanType == 'ASL')
 %
 % OUTPUT:
 %   x                   - same as input
 %
 % -----------------------------------------------------------------------------------------------------------------------------------------------------
-% DESCRIPTION: This function collects QC parameters for a module.
+% DESCRIPTION: This function collects QC parameters for a module for a given subject and for ASL for a given subject+session
 % 
-% EXAMPLE: x = xASL_qc_CollectParameters_Structural(x, 10, 'ASL');
+% EXAMPLE: x = xASL_qc_CollectParameters_Structural(x, 10, 'ASL', 1);
```

Comment:
Perhaps create more than a single example then. If you don't add 1, it will default to 1 right?

Reply:
Agree. Done in c857e9b4